cl/lib/elasticsearch_utils.py

```diff
@@ -2769,7 +2769,15 @@ def build_semantic_query(
     # This enables hybrid search by combining keyword and semantic results
     if keyword_query:
         semantic_query.extend(
-            build_fulltext_query(fields, keyword_query, only_queries=True)
```

Comment:
Now that the `fields` variable is unused, can we remove it from the method signature?

Reply:
Sure!